pkg/l2serviceattachmentparser/l2serviceattachmentparser.go

```diff
@@ -58,6 +66,41 @@ func (sattp *L2SrvAttParser) ParseL2ServiceAttachment(d *render.RenderData) (str
 		return "", err
 	}
 
+	if len(sattp.subnetResources) > 0 {
+		if len(sattp.subnetResources) > 2 {
```

Comment:
Maybe we can remove this check because we already have this validation check here:
https://github.com/Nordix/eno/blob/eno-subnet-routes/api/v1alpha1/l2service_types.go#L34

I think if we keep it is just redundant.

Reply:
Ok, removed the check.